lib/plan-finder.js

```diff
@@ -259,10 +266,56 @@ class PlanFinder {
         if (!this.isMimeType(outputInstructions.type)) {
             throw new RenditionFormatUnsupportedError(`outputInstructions.type (${outputInstructions.type}) is not propertly formatted`);
         }
-        const plan = this.BFS(source, outputInstructions);
+
+        let plan;
+        // get image metadata and check for "Orientation" property
+        const metadata = await this.getImageMetadata(source.path);
+        // if image has Orientation metadata set image to sensei compliant orientation
+        if (metadata.Orientation) {
+            let tempPlan = new Plan();
+            // make instructions for orientation
+            const instructions = {input: source, output: { type: source.type }};
+            tempPlan.add("workerCallback-worker-flite", instructions);
```

Comment:
the workerCallback available will not always be `worker-flite`. We could be inside worker-pie